test/ulid/rails_test.rb

```diff
@@ -51,14 +51,11 @@ def test_saves_when_ulid_foreign_key_is_nil
   end
 
   def test_has_many_through
-    # Doesn't work until https://github.com/rails/rails/issues/35839 is released
-    #    user = User.create!
-    #    article = user.articles.create!
-    #    user.reload
-    #
-    #    binding.irb
-    #
-    #    assert user.articles.count == 1
-    #    assert user.articles[0] == article
+    user = User.create!
+    article = user.articles.create!
+    user.reload
+
+    assert user.articles.count == 1
+    assert user.articles[0] == article
```

Comment:
```suggestion
    assert_equal user.articles.first, article
```

Reply:
Thanks @HeyNonster , these are great suggestions! I'll do a deeper code style changes in a later PR. I just want this diff to look like just an uncommented test that pass now.